src/main/java/com/puppycrawl/tools/checkstyle/utils/ModuleReflectionUtils.java

```diff
@@ -76,17 +78,36 @@ public static boolean isCheckstyleModule(Class<?> clazz) {
                     || isFilterModule(clazz)
                     || isFileFilterModule(clazz)
                     || isTreeWalkerFilterModule(clazz)
+                    || isAuditListener(clazz)
```

Comment:
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/DefaultLogger.java#L44

`public class DefaultLogger extends AutomaticBean implements AuditListener { `

it might be used in config file only due to `extends AutomaticBean` .
This become a nuance , as always thought about config as way to define how validation should be done, but not way to define destination of output. That was a tasks of caller, CLI, plugin, application .... .  
It is good question if we should continue to support this. I need to mediate about this concept for some time to make a decision is good or bad for checkstyle.